src/main/Tree.razor.cs

```diff
@@ -60,28 +60,26 @@ private async Task HandleSelectionOptionChanged(ContextMenuOption value)
                     this.EditNeuron = this.SelectedNeuron.Neuron;
                     break;
                 case ContextMenuOption.ExpandUntilPostsynapticExternalReferences:
-                    this.ShowExpandModal();
-                    this.SelectedNeuron.ConfigureExpansionTimer(
-                        this.expansionTimer,
-                        ExpansionType.PostsynapticUntilExternalReferences,
-                        this.pluginSettingsService.ExpandTimeLimit,
-                        this.ExpansionTimer_Elapsed
-                    );
-                    this.SelectedNeuron.StartExpansionTimer();
+                    this.Expansion(ExpansionType.PostsynapticUntilExternalReferences);
```

Comment:
Good refactoring effort, however ensure method names use verb names instead of nouns eg. "ExpandSelectedNeuron"